CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
+## 8.17.0
+  - Compatible with stack (Logstash and Elasticsearch) 8.17 versions [#212](https://github.com/elastic/logstash-filter-elastic_integration/pull/212)
```

Comment:
Just aligning the changelog entry with https://github.com/elastic/logstash-filter-elastic_integration/pull/210/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR2-R3

```suggestion
  - Aligns with stack major and minor versions, no functional changes [#210](https://github.com/elastic/logstash-filter-elastic_integration/pull/210)
  - Compatible with Logstash and Elasticsearch 8.17 versions
```

Reply:
Just aligning the changelog entry with https://github.com/elastic/logstash-filter-elastic_integration/pull/210/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR2-R3

```suggestion
  - Aligns with stack major and minor versions, no functional changes [#210](https://github.com/elastic/logstash-filter-elastic_integration/pull/210)
  - Embeds Ingest Node components from Elasticsearch 8.17
  - Compatible with Logstash 8.x
```

Reply:
The final suggestion changes `Compatible with Logstash 8.x` to `Compatible with Logstash 8.15+`
```suggestion
    - Aligns with stack major and minor versions, no functional changes [#212](https://github.com/elastic/logstash-filter-elastic_integration/pull/212)
  - Embeds Ingest Node components from Elasticsearch 8.17
  - Compatible with Logstash 8.15+
```